src/comp.py

```diff
@@ -23,54 +23,58 @@ def __repr__(self):
 # whose name starts with 'D':
 
 print("Starts with D:")
-r = [char.name for char in humans]  # TODO
+r = [char.name for char in humans]  
 d_list = [word for word in r if word[0] == "D"]
 print(d_list)
 
 # Write a list comprehension that creates a list of names of everyone
 # whose name ends in "e".
 
 print("Ends with e:")
-r = [char.name for char in humans]  # TODO
+r = [char.name for char in humans]  
 e_end = [word for word in r if word[-1] == 'e']
 print(e_end)
 
 # Write a list comprehension that creates a list of names of everyone
 # whose name starts with any letter between 'C' and 'G' inclusive.
 
 print("Starts between C and G, inclusive:")
-r = [char.name for char in humans]  # TODO
+r = [char.name for char in humans]  
 first_letters = ["C", "D", "E", "F", "G"]
 cg_names = [name for name in r if (name[0] in first_letters)]
 print(cg_names)
 
 # Write a list comprehension that creates a list of all the ages plus 10.
 print("Ages plus 10:")
-r = [peeps.age for peeps in humans]  # TODO
+r = [peeps.age for peeps in humans]  
 ten = [age * 10 for age in r]
 print(ten)
 # Write a list comprehension that creates a list of strings which are the name
 # joined to the age with a hyphen, for example "David-31", for all humans.
 print("Name hyphen age:")
 r = []  # TODO
+for human in humans:
+    r.append(human.name+"-"+str(human.age))
 print(r)
 
 # Write a list comprehension that creates a list of tuples containing name and
 # age, for example ("David", 31), for everyone between the ages of 27 and 32,
 # inclusive.
 print("Names and ages between 27 and 32:")
-r = [peeps.age for peeps in humans if (peeps.age < 32) & (peeps.age > 27)]  # 
+r = [peeps.age for peeps in humans if (peeps.age < 32) & (peeps.age > 27)]  
 print(r)
 
 # Write a list comprehension that creates a list of new Humans like the old
 # list, except with all the names capitalized and the ages with 5 added to them.
 # The `humans` list should be unmodified.
 print("All names capitalized:")
-r = [char.name for char in humans]  # TODO
-upper = [word.upper() for word in r]
-print(upper)
+r = []
+for human in humans:
+    r.append((str.upper(human.name),(human.age+5)))
+print()
```

Comment:
Two things. For the string .upper() method, you put it at the end of the string you want to be capitalized like so: 
```
r.append((human.name.upper(), human.age+5))
```
Also, you forgot to pass an argument to print on line 74.